packages/controls/src/controls/group/group.test.types.ts

```diff
@@ -1,19 +1,19 @@
 import { expectTypeOf } from 'expect-type'
 
-import { ControlDataTypeKey } from '../../../common'
+import { ControlDataTypeKey } from '../../common'
 
 import {
   type DataType,
   type ResolvedValueType,
   type ValueType,
-} from '../../associated-types'
-import { Color } from '../../color'
-import { List } from '../../list'
-import { Number } from '../../number'
+} from '../associated-types'
+import { Color } from '../color'
+import { List } from '../list'
+import { Number } from '../number'
 
-import { ShapeV2, ShapeV2Definition } from './shape-v2'
+import { Group, GroupDefinition } from './group'
```

Comment:
Other than renaming, type tests are mostly the same. Highlighting noteworthy changes below.